src/gui_impl/main_window.cpp

```diff
@@ -44,7 +44,7 @@ MainWindow::MainWindow(QSharedPointer<ApplicationContext> application_context)
     setUnifiedTitleAndToolBarOnMac (true);
     
     QSettings settings("SigViewer");
-    resize(settings.value("MainWindow/size", QSize(900, 800)).toSize());
+    resize(settings.value("MainWindow/size", QSize(1500, 800)).toSize());
```

Comment:
Let's not make this value too large. After all, there might be people with low-resolution displays. Everyone else can change the window size anyway and it will be restored the next time. I suggest to use a value of 1000 here.